src/hotspot/share/services/memReporter.cpp

```diff
@@ -22,20 +22,22 @@
  *
  */
 #include "precompiled.hpp"
+#include "jfr/jfrEvents.hpp"
```

Comment:
Not sure I agree, I see the `MemReporter` as the reporting layer and it now knows about JFR to be able to report through it.

We have in other places created small helpers that send the events to avoid including `jfrEvents.hpp` all over the place, and we could to that here as well. But I don't see a problem with having the reporter residing along side the other reporters.

Reply:
Okay.